### PR TITLE
Fix issue #29 - Problem on fs.copyFileSync when publishing files

### PR DESCRIPTION
**Pull request by rcmsjr**
Fix issue on fs.copyFileSync when publishing files to destination directory with node v8.5.0

### CODE REVIEW

**Comment by ricardofadini** [comment]
+1

**Comment by roberto-souza-dbc** [comment]
+1

**Comment by hwguilherme** [comment]
+1

File: publish.js [hunk 1]
```diff
@@ -607,8 +607,9 @@ exports.publish = function(taffyData, opts, tutorials) {
 
     staticFiles.forEach(function(fileName) {
         var toDir = fs.toDir( fileName.replace(fromDir, outdir) );
+				var toFileName = path.basename(fileName);
```
Comment (minkyu-yi) [inline publish.js]: Could you fix the indentation with spaces ? (no tabs please) :)

File: publish.js [hunk 2]
```diff
@@ -633,8 +634,9 @@ exports.publish = function(taffyData, opts, tutorials) {
             extraStaticFiles.forEach(function(fileName) {
                 var sourcePath = fs.toDir(filePath);
                 var toDir = fs.toDir( fileName.replace(sourcePath, outdir) );
+								var toFileName = path.basename(fileName);
```
Comment (minkyu-yi) [inline publish.js]: Also in here indentation with spaces :)

Comment (rcmsjr) [reply publish.js]: Ok, I'm updating.

**Comment by minkyu-yi** [comment]
Thank you!!! I'll release this asap!! 👍

**Comment by minkyu-yi** [comment]
Revert

The `fs` module used is in jsdoc. Not in native node.js v8.5.

So We should update jsdoc versions to 3.5.5.

**Pull request merged**